university-of-south-australia-2017-harvard.csl

```diff
@@ -267,7 +265,7 @@
     <label variable="page" form="short" suffix=" "/>
     <text variable="page"/>
   </macro>
-  <macro name="access">
+  <macro name="access-date">
```

Comment:
I didn't look at this in detail, but it looks like not all item types that get a URL should get an access date (see e.g. the e-book example) so maybe this needs to be restricted (to blogposts and webpages perhaps?)

Reply:
Almost all types with a URL should have an access date. The exceptions will fall into the types discussed above where we don't want to show the URL anyway. I've repositioned the logic for these types to make it clear that they affect both the access date and the URL.